content/guides/embed/ui-elements/explorer-metadata.md

```diff
@@ -0,0 +1,193 @@
+---
+rank: 4
+related_endpoints:
+  - get-metadata-templates-id
+related_guides:
+  - embed/ui-elements
+required_guides:
+  - embed/ui-elements/installation
+related_resources: []
+alias_paths:
+  - /docs/box-content-explorer
+  - /docs/content-explorer
+---
+
+# Content Explorer - metadata view
+
+With Content Explorer you can also display files and folders
+based on their metadata.
+The metadata view uses [metadata template][template] and [metadata query][metadata-query] to
+find the data you want to display.
+
+![Metadata view](./images/explorer-view.png)
+
+## Prerequisites
+
+* Read the [Content Explorer][explorer] guide.
+* Check the [metadata terminology][terminology].
+* Check the information on [metadata queries][metadata-query].
+
+## Create and configure an app
+
+1. [Create a Box app][box-app].
+2. Add the local development address in the CORS Domains: ![CORS Domains](./images/box-app-cors.png)
+3. Generate a [developer token][token].
+
+## Create a metadata template
+
+The next step is to create a metadata template you will use to populate the Content Explorer.
+
+1. Create a metadata template. You can use [Metadata API][creating-templates-api] or [Admin Console][creating-templates-ui] to do so.
+2. Apply an already created template to a Box folder. Make sure you enable the
```

Comment:
You can also apply to metadata template directly to a file

Reply:
Added a note below the steps.